main/pom.xml

```diff
@@ -137,6 +137,31 @@
                     <exclude>META-INF/*.RSA</exclude>
                   </excludes>
                 </filter>
+                <filter>
+                  <!-- org.apache.xerces.jaxp.DocumentBuilderFactoryImpl used in spark-yarn -->
```

Comment:
this part wasn't yarn specific. fixed in master

Reply:
ok, removed